certified-connectors/Actsoft/apiDefinition.swagger.json

```diff
@@ -483,7 +497,12 @@
         ],
         "responses": {
           "200": {
-            "description": "Client deleted."
+            "description": "Client has been updated.",
+            "schema": {}
```

Comment:
@rdameron Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger. 

It might receive a response like "success" or "failed" for the delete or update function. Please have at least one schema definition like "string": "Success"

Reply:
The same comment applies to all the empty schema definitions.

Reply:
This has been resolved